iroha_network/src/lib.rs

```diff
@@ -281,6 +281,7 @@ mod tests {
     }
 
     #[tokio::test(flavor = "multi_thread")]
+    #[ignore]
```

Comment:
i guess you can just remove iroha_network from repo

Reply:
There is a `mock` feature, and `Torii` depends on it.

Reply:
Which one?

Reply:
For this we have separate issue: https://github.com/hyperledger/iroha/issues/1382